src/Knp/DictionaryBundle/Form/Type/DictionaryType.php

```diff
@@ -42,14 +43,6 @@ public function setDefaultOptions(OptionsResolverInterface $resolver)
      */
     public function getParent()
     {
-        return 'choice';
-    }
-
-    /**
-     * {@inheritdoc}
-     */
-    public function getName()
-    {
-        return 'dictionary';
+        return ChoiceType::class;
```

Comment:
You should not use the `::class` notation but write the full class name here to keep the compatibility with php 5.4.* (since symfony 2.8 supports this php version)